.vscode/settings.json

```diff
@@ -0,0 +1,40 @@
+{
+    "editor.formatOnSave": false,
+    "cmake.configureOnOpen": true,
+
+    "[diff]": {
+        "files.trimTrailingWhitespace": false
+    },
+
+    // uncomment this to compile 3rd parties
+    // "cmake.sourceDirectory": "${workspaceFolder}/dynawo/3rdParty",
+    // "cmake.buildDirectory": "${workspaceFolder}/build-code/3rdParty",
```

Comment:
If one day one would want to deploy from what you build in vscode, you would need to put the build in ${workspaceFolder}/build-code/3rdParty/build (it is a fixed convention used in the deploy script)